web/src/search.ts

```diff
@@ -77,6 +77,11 @@ function narrow_or_search_for_term({on_narrow_search}: {on_narrow_search: OnNarr
     return get_search_bar_text();
 }
 
+function focus_search_input_at_end(): void {
+    $("#search_query").trigger("focus");
+    window.getSelection()!.modify("move", "forward", "line");
```

Comment:
This probably deserves a bit of a comment explaining what you're doing.

Reply:
Just added a short comment and pushed. Let me know if this level of description is what you meant or if you wanted more detail

Reply:
Sometimes when we called `$("#search_query").trigger("focus");` before, the cursor was being placed at the start of the input text, so now we ensure it's always at the end. I have a bit of fog-brain and would probably benefit from better name suggestions for this function 😅